packages/framework/src/hooks/useEvaluate.ts

```diff
@@ -63,5 +64,18 @@ export const useEvaluate = <T extends Record<string, unknown>>(
     return result;
   }, [values, translate]);
 
-  return merge({}, values, bindings, translatedkeys);
+  // evaluate flutter color hex codes
+  const hexCodes = useMemo(() => {
+    const hexCodesMaps: string[][] = [];
+    findHexCodes(values, [], hexCodesMaps);
+
+    const result = {};
+    hexCodesMaps.forEach(([name, value]) => {
+      set(result, name, value);
+    });
+
+    return result;
+  }, [values]);
```

Comment:
This is only going to work if the hexcode is hardcoded but not if its defined in a theme file or comes from the result of an expression. I think you should do the conversion after the bindings have been evaluated and only do it on the styles object.